README.md

```diff
@@ -58,20 +62,48 @@ fvm flutter run
 
 ### Contributing
 
 This project follows a conventional branch structure and [Conventional Commits](https://www.conventionalcommits.org/). The stable branch is `main`, which shouldn't be directly commited to. The main development branch, `dev`, is merged into `main` after checks pass, but `dev` also shouldn't be directly commited to. Commits should be made into branches branched out from the `dev` branch, e.g. `feat/my-awesome-feature`.
 
-To contribute, open a pull request from your branch to the `dev` branch. For better versioning, make sure to use Conventional Commits, perhaps with the help of [Commitizen](https://commitizen-tools.github.io/commitizen/). For major changes, please open an issue first to discuss what you would like to change.
+To contribute, open a pull request from your branch to the `dev` branch, naming your pull request according to Conventional Commits and including a changelog in the description (see below). Ideally, your commits should also follow Conventional Commits, perhaps with the help of [Commitizen](https://commitizen-tools.github.io/commitizen/). For major changes, please open an issue first to discuss what you would like to change.
+
+All changes are documented in `CHANGELOG.md`, which is generated with the help of [git-cliff](https://github.com/orhun/git-cliff/). After installing git-cliff and commiting your last changes to your PR, simply run the following command to update the changelog (and then commit this as well!)
+
+```bash
+git cliff --sort newest --output CHANGELOG.md
+```
 
-Please make sure to update tests as appropriate.
+For the description of your PR, generate the changelog from the latest commit before your commits, which can be generated using the hash of this latest commit, for example
+
+```bash
+git cliff f5d5f263870b559e347824700956646501ae48ff..
+```
+
+This should also be the message body of the merge commit of the PR, with the commit title being the name of the PR (in Conventional Commit format).
+
+_TODO Add testing instructions once a testing suite is implemented_
+
+<!--
+Probably using bloc_test and mockito or mocktail.
+-->
 
 ## Authors and Acknowledgements
 
-This project was forked from the [original](https://github.com/choonkiatlee/ICRISAT-mobile) by [Choon Kiat Lee](https://github.com/choonkiatlee) and it was taken on by [Greg Chu](https://github.com/gregchu6) & [Juan Rodgers](https://github.com/rodgersjuan).
+Gibsonify is being developed by [Shazril Suhail](https://github.com/sshazril) and [Adam Sroka](https://adamsroka.io), who have taken on the previous work of [Greg Chu](https://github.com/gregchu6), [Juan Rodgers](https://github.com/rodgersjuan), and [Choon Kiat Lee](https://github.com/choonkiatlee).
+
+The development is being supported by the [Centre for Global Equality](https://centreforglobalequality.org) and the [University of Cambridge](https://www.cam.ac.uk).
```

Comment:
We'll need include Lara and me as supervisors, and acknowledge the collaboration with ICRISAT too. I can get that side sorted once we have accepted the pull request.

Reply:
Yes sure, or you can just comment here the exact phrasing that you'd like, and I can commit that. Or I can add you as an assignee to this PR and you can commit the changes as well 😊.